job_board/tests.py

```diff
@@ -0,0 +1,62 @@
+from job_board.models.job_type import JobType
+from job_board.models.location import Location
+from job_board.models.years_of_experience import YearsOfExperience
+from job_board.models.preference import Preference
+import pytest
+
+
+@pytest.mark.django_db
+class TestJobBoardModels:
+
+    def test_data_migration(self):
+        assert JobType.objects.filter(text='Web developer').exists()
+        assert Location.objects.filter(name='Haifa', latitude=32.804135934085565, longitude=34.98975968417104).exists()
+        assert YearsOfExperience.objects.filter(text='no experience').exists()
+
+    def test_data_creation(self):
```

Comment:
If it is test_data_creation,
consider using fixtures, which are holders for new tests data

And then assert the content against the fixtures data